src/lib/invite.js

```diff
@@ -202,41 +161,109 @@ async function claimPointFromInvite({
     false
   );
   keysTx.gas = 150000;
-  keysTx.nonce = 1;
 
-  //TODO  refactor this process into function
+  // configure management proxy
+
+  let managementTx = azimuth.ecliptic.setManagementProxy(
+    contracts,
+    point,
+    wallet.management.keys.address
+  );
+  managementTx.gas = 200000;
+
+  // set spawn & voting proxies situationally
+
+  let txs = [transferTmpTx, keysTx, managementTx];
+  if (
+    azimuth.azimuth.getPointSize(point) !== azimuth.azimuth.PointSize.Planet
```

Comment:
`azimuth.azimuth.getPointSize(point)` will return a Promise (right?) which means we should probably `await` it here or we'll always get `false`

Reply:
Nope, [this is just a local mirror of the on-chain logic](https://github.com/urbit/azimuth-js/blob/master/azimuth.js#L333).